asserts/database.go

```diff
@@ -57,6 +86,7 @@ type consistencyChecker interface {
 // Database holds assertions and can be used to sign or check
 // further assertions.
 type Database struct {
+	be          Backstore
```

Comment:
Can we please use "bs" rather than "be" as the abbreviation? "be" is such a common verb, and doesn't enter my mind as "backstore".